README.md

```diff
@@ -147,6 +147,21 @@ $model->published_at = Timezone::date($request->published_at);
 
 **This is not a bug**, it is intended behavior since one should be fully aware of the Carbon instance's timezone before assigning it.
 
+### Edge cases
+
+If you need to use the `TimezonedDatetime` cast on the timestamp columns (`created_at` and/or `updated_at`) AND you're expecting to handle dates with timezones other than UTC or the one you've defined with `Timezone::set()`, you will need to apply the `Whitecube\LaravelTimezones\Concerns\HasTimezonedTimestamps` trait on your model.
```

Comment:
```suggestion
If you need to use the `TimezonedDatetime` or `ImmutableTimezonedDatetime` casts on the default timestamp columns (`created_at` and/or `updated_at`) AND you're expecting to handle dates with timezones other than UTC or the one you've defined with `Timezone::set()`, you will need to apply the `Whitecube\LaravelTimezones\Concerns\HasTimezonedTimestamps` trait on your model.

```php
use Illuminate\Database\Eloquent\Model;
use Whitecube\LaravelTimezones\Concerns\HasTimezonedTimestamps;

class Post extends Model
{
    use HasTimezonedTimestamps;

    /**
     * The attributes that should be cast.
     */
    protected $casts = [
        'created_at' => ImmutableTimezonedDatetime::class,
        'updated_at' => ImmutableTimezonedDatetime::class,
    ]
}
```

```